pkg/etcdcli/health.go

```diff
@@ -7,10 +7,29 @@ import (
 	"sync"
 	"time"
 
+	"github.com/prometheus/client_golang/prometheus"
 	"go.etcd.io/etcd/etcdserver/api/v3rpc/rpctypes"
 	"go.etcd.io/etcd/etcdserver/etcdserverpb"
+	"k8s.io/component-base/metrics/legacyregistry"
 )
 
+func init() {
+	legacyregistry.RawMustRegister(raftTerms)
+}
+
+const raftTermsMetricName = "etcd_raft_terms_total"
```

Comment:
nit: should we rename this?

`etcd_debugging_raft_terms_total`

Per upstream

```
Note that any etcd_debugging_* metrics are experimental and subject to change.
```
This gives us the option to change remove if we feel so fit. I think any new metric should be considered experimental through one release.

Reply:
Good call, fixed